buildSrc/src/main/kotlin/otel.java-conventions.gradle.kts

```diff
@@ -67,9 +67,7 @@ tasks {
 
     testLogging {
       exceptionFormat = TestExceptionFormat.FULL
-      showExceptions = true
-      showCauses = true
-      showStackTraces = true
```

Comment:
removed these because they all default to `true` (and to match Java instrumentation repo)